ClassLibrary/ClassLibrary/Person.cs

```diff
@@ -0,0 +1,158 @@
+﻿/* Person.cs creates the IPerson interface and creates entity classes that implement it in order to represent the data of the student
+ * and professors who will use the system. Both students and professors have name, net ID, UTD ID, and password variables. Students,
+ * in addition to these, have variables for their class name and the group they're in. Professors have a list variable that contains 
+ * the classes they are incharge of. The name, net ID, UTD ID, class names, and group numbers can be modified with regular get/set
+ * methods. Passwords can be set but can't be retrieved; a password must be provided in order to check that it matches the Person's
+ * password. Professor contains methods for adding class(es), removing class(es), retrieving classes, and checking if a certain
+ * class is already in their class list.
+ * Author:  Jesus Barrera-Gilabert III
+ * Date:    10/11/2024
+ * Class:   Computer Science Project CS 4485.0W1
+ * Net ID:  jxb171030
+ * UTD ID:  2021348532
+ * Version: 0.1
+ */
+using System;
+using System.Collections.Generic;
+
+//Interface used by Student and Professor
+interface IPerson
+{
+    string Name { get; set; }
+    int UtdID { get; set; }
+    string NetID { get; set; }
+
+    //Change password to pass
+    void ChangePassword(string pass);
+    //Check if pass matches Password; return true if it does
+    bool CheckPassword(string pass);
+}
+
+//Represents the data of a student with operations for retrieving or editing it
+class Student : IPerson
+{
+    private string _password;
+
+    //Constructor
+    public Student(string name, int utdID, string netID, string cName, int group)
+    {
+        Name = name;
+        UtdID = utdID;
+        NetID = netID;
+        _password = netID;

```

Comment:
Within the Class diagram password is a string that id default utdID.
    in constructor of student and professor, password = netID

Reply:
that was an error, it was supposed to default to net id

Reply:
no problem, i changed it to netID